R/github_document.R

```diff
@@ -14,6 +14,7 @@
 #'   locally previewing what the document will look like on GitHub.
 #' @param keep_html \code{TRUE} to keep the preview HTML file in the working
 #'   directory. Default is \code{FALSE}.
+#' @param ext Extention of the output document (defaults to ".md")
```

Comment:
```suggestion
#' @inheritParams md_document
```

would be better I guess.